src/rdkafka.c

```diff
@@ -956,33 +1023,40 @@ static RD_INLINE void rd_kafka_stats_emit_toppar (char **bufp, size_t *sizep,
 		   rktp->rktp_hi_offset,
                    consumer_lag,
                    rd_atomic64_get(&rktp->rktp_c.tx_msgs),
-		   rd_atomic64_get(&rktp->rktp_c.tx_bytes),
-		   rd_atomic64_get(&rktp->rktp_c.msgs),
+                   rd_atomic64_get(&rktp->rktp_c.tx_msg_bytes),
+                   rd_atomic64_get(&rktp->rktp_c.rx_msgs),
+                   rd_atomic64_get(&rktp->rktp_c.rx_msg_bytes),
+                   rk->rk_type == RD_KAFKA_PRODUCER ?
```

Comment:
would indent differently, personally i like:
```
rk->rk_type == RD_KAFKA_PRODUCER
    ? rd_atomic64_get(&rktp->rktp_c.producer_enq_msgs)
    : rd_atomic64_get(&rktp->rktp_c.rx_msgs)
```

Reply:
It follows the librdkafka coding style, fwiw.

Reply:
sounds good, fwiw my complaint is that at a glance, you can't see the parameters distinctly.